docs/src/literate/heat_equation.jl

```diff
@@ -127,20 +127,25 @@ function doassemble(cellvalues::CellScalarValues{dim}, K::SparseMatrixCSC, dh::D
     f = zeros(ndofs(dh))
     assembler = start_assemble(K, f)
 
+    celldofs = zeros(Int, ndofs_per_cell(dh))
+    cellcoords = zeros(Vec{dim,T}, JuAFEM.nnodes_per_cell(dh))
+
     # It is now time to loop over all the cells in our grid. We do this by iterating
     # over a `CellIterator`. The iterator caches some useful things for us, for example
     # the nodal coordinates for the cell, and the local degrees of freedom.
     #+
-    @inbounds for cell in CellIterator(dh)
```

Comment:
Does `CellIterator` not work for `MixedDofHandler`?

Reply:
Not at the moment, but could quite easily fix that so it works for both dofhandler and mixeddofhandler.